apps/explorer/src/ui/Link.tsx

```diff
@@ -19,6 +19,10 @@ const linkStyles = cva(
             uppercase: {
                 true: 'uppercase',
             },
+            size: {
+                md: '!text-body',
```

Comment:
Why important?

Reply:
Maybe we should name these `body` and `bodySmall` to align with the text sizes? I don't feel strongly here though.

Reply:
I do not think the component is fully flushed out yet. This just allows us to match the font size of the address link in the sender-recipient component without affecting the other use case. We can revisit this in another PR.